commands/sync.go

```diff
@@ -0,0 +1,62 @@
+package commands
+
+import (
+	"github.com/bndw/pick/backends"
+	"github.com/bndw/pick/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	rootCmd.AddCommand(&cobra.Command{
+		Use:   "sync [other-pick-safe]",
+		Short: "Sync current safe with another pick safe",
+		Long:  "The sync command is used to sync the current pick safe with another pick safe.",
```

Comment:
It doesn't just import other accounts, it also sync existing accounts, but only if both pick safes were created at the same time (i.e. are the same pick safes, which might have different accounts / account histories in it). The current implementation doesn't allow to import accounts from a completely different safe.

Note syncing is not implemented yet.